### PR TITLE
Update german.xml to v7.9.6

### PR DESCRIPTION
**Pull request by schnurlos**
Following 926e6e97d82b3d63103cb2297b372af5cd469ca6, 6834d796ab553438827fe7e07a1e4caef8460f6f & 874f0d01401e7c73b9965294cec41f3245acfc5a

### CODE REVIEW

File: PowerEditor/installer/nativeLang/german.xml [hunk 1]
```diff
@@ -1369,6 +1381,7 @@ Finde in allen Dateien außer exe, obj &amp;&amp; log:
 			<finder-select-all value="Alles wählen"/>
 			<finder-clear-all value="Alles löschen"/>
 			<finder-open-all value="Alles öffnen"/>
+			<finder-purge-for-every-search value="Für jede Suche neu eingeben"/>
```
Comment (MarkusBodensee) [inline PowerEditor/installer/nativeLang/german.xml:1384]: @schnurlos
Hmm I think this translation could be misleading. Maybe it is better to write: "Vor jeder Suche leeren" or something similar?

Comment (schnurlos) [reply PowerEditor/installer/nativeLang/german.xml:1384]: @MarkusBodensee: confirmed - ich werde das anpassen, Danke für den Tip.

**Comment by donho** [comment]
@schnurlos 
Could you please check the latest report of the old localization file vs new localization file to make sure if your PR updated?
https://github.com/notepad-plus-plus/notepad-plus-plus/issues/9854

Thank you!

**Comment by schnurlos** [comment]
@donho : done

**Pull request closed without merging**